### PR TITLE
Update Mojave/README.md

### PR DESCRIPTION
**Pull request by stereoplegic**
Add notes for changing Clover resolution and confirm working with Ubuntu 19.04

### CODE REVIEW

File: Mojave/README.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ Note: Use `create_iso_mojave.sh` for generating a macOS Mojave based ISO image.
 
 Note: Use 64-bit Ubuntu 18.04.2 LTS as the host for "best" results.
 
+Note: Tested working on Ubuntu 19.04 host.
+
+With either method, edit the desired Clover screen resolution in `./config.plist.stripped.qemu` (and set to match in EDK II OVMF UEFI, or else bad things happen when resolution changes on Clover startup in fullscreen).
```
Comment (kholia) [inline Mojave/README.md]: What is this 'either method' in this context?

Comment (stereoplegic) [reply Mojave/README.md]: "either method" refers to the old or new method of generating Clover disk mentioned in the same document.

Comment (pmetzger) [reply Mojave/README.md]: You probably need to edit the document to make that clear.

Comment (kholia) [reply Mojave/README.md]: Thanks. Documented at https://github.com/kholia/OSX-KVM/blob/master/notes.md#higher-resolution-uefi--clover now.

Comment (stereoplegic) [reply Mojave/README.md]: Thanks. Sorry for the silence, busy weekend. I'll revisit this to see if I can further clarify if I find the time.

**Pull request closed without merging**